### PR TITLE
View deployment's checkpoint id and end time

### PR DESCRIPTION
**Pull request by pigbug419**
### Changes
- `pf deployment view` shows model ckpt id and end time
- Fix bug in `deployment list` to list only current project's deployment.

### CODE REVIEW

**Review by kdh0102 — APPROVED** [review]
LGTM

File: pfcli/deployment.py [hunk 1]
```diff
@@ -234,6 +236,9 @@
 )
 deployment_table.add_substitution_rule("Terminated", "[bold]Terminated[/bold]")
 
+deployment_org_table.add_substitution_rule(
+    "Pending", "[bold yellow]Pending[/bold yellow]"
+)
```
Comment (kooyunmo) [inline pfcli/deployment.py:241]: By the PR https://github.com/friendliai/periflow-cli/pull/165, the "Pending" status is deprecated.

**Review by kooyunmo — CHANGES_REQUESTED** [review]
I left a minor comment. Otherwise, LGTM!

File: pfcli/deployment.py (hunk 1 above)
Comment (pigbug419) [reply pfcli/deployment.py:241]: Thank you for the remind!

**Comment by pigbug419** [comment]
@kooyunmo Thank you! I've addressed your comment!

**Review by kooyunmo — APPROVED** [review]

**Pull request merged**